.circleci/config.yml

```diff
@@ -3,7 +3,7 @@ version: 2
 defaults: &defaults
   working_directory: ~/repo
   docker:
-    - image: circleci/node:12
+    - image: circleci/node:14
```

Comment:
This is unrelated but I am pushing for all of our modules to be built on node 14 🙂